onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -417,7 +423,9 @@ TEST(SplitOperatorTest, SplitAttributeSumTooSmall) {
   outputs.push_back({{1, 2}, {1.f, 2.f}});
   outputs.push_back({{2, 2}, {3.f, 4.f, 5.f, 6.f}});
 
-  RunTest<float>(axis, splits, input, outputs, false, true, "Cannot split using values in 'split' attribute");  //TensorRT parser: Assertion failed: axis != BATCH_DIM
+  RunTest<float>(axis, splits, input, outputs, false, false, true, "Cannot split using values in 'split' attribute");
+  //TensorRT parser: Assertion failed: axis != BATCH_DIM
+  //Nuphar returns FuncKernel call failed with error code: -1
 }
```

Comment:
Is this expected or should it be fixed at some point? Would be good to clarify whether this test is disabled temporarily or not, and ideally say why it returns error code -1.

Reply:
The Nuphar side failure might be already fixed with recent FuncKernel changes. Will revert the line.

---
In reply to: [312429916](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312429916) [](ancestors = 312429916)